src/cli_util.rs

```diff
@@ -1417,6 +1418,22 @@ pub fn resolve_base_revs(
     }
 }
 
+fn add_multi_message_to_hint_if_mutliple_revisions(err: CommandError) -> CommandError {
+    if let CommandError::UserError { message, hint } = err {
+        let hint = if message.contains("more than one revision") {
```

Comment:
This is not the most robust test, but I'm hoping that since this is interior to the `cli_util` module it is OK.

I also tried making a custom type for this, but it's a bit messy. See https://github.com/ilyagr/jj/tree/multi-match.

Suggestions welcome :).